deploy/hub-of-hubs-spec-transport-bridge.yaml.template

```diff
@@ -71,6 +71,14 @@ spec:
                 secretKeyRef:
                   name: hub-of-hubs-database-transport-bridge-secret
                   key: url
+            - name: HOH_TRANSPORT_TYPE
+              value: "$HOH_TRANSPORT_TYPE"
+            - name: KAFKA_PRODUCER_ID
+              value: "hoh-spec-producer"
+            - name: KAFKA_PRODUCER_HOSTS
+              value: "$KAFKA_PRODUCER_HOSTS"
```

Comment:
see previous comments, this can be set to the cluster internal service, no need for setting env var.